rpm/opensuse.go

```diff
@@ -41,7 +45,10 @@ func (b *OpenSUSEBackend) GetKernelHeaders(directory string) error {
 	}
 	err := b.dnfBackend.GetKernelHeaders(pkgNevra, directory)
 	if err == nil {
-		return nil
+		b.logger.Infof("successfully downloaded %s", pkgNevra)
+
+		// kernel-[flavour]-devel packages require kernel-devel
+		return b.dnfBackend.GetKernelHeaders("kernel-devel", directory)
```

Comment:
This downloads the extra package if the limited repo set is successful. Do we also need to download this package after expanding to all repos?

Reply:
Right, but this branch is the success case for the limited repo set. If this initial attempt fails, it expands to all repos. Then if that download *works*, it will NOT download `kernel-devel`.

Reply:
I added a suggestion, showing what I mean.

Reply:
Ah, I misunderstood what you meant. You're right, we need to download kernel-devel if it's found in a different repo as well